app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PiecesController < ApplicationController
+	def create
```

Comment:
I don't think you'll need a `create` method. All of the piece creation should be handled within the game creation. I don't think you would need to create a new piece using a controller